Mantle/NSValueTransformer+MTLPredefinedTransformerAdditions.m

```diff
@@ -77,25 +96,7 @@ + (void)load {
 
 		[NSValueTransformer setValueTransformer:URLValueTransformer forName:MTLURLValueTransformerName];
 
-		MTLValueTransformer *booleanValueTransformer = [MTLValueTransformer
-			transformerUsingReversibleBlock:^ id (NSNumber *boolean, BOOL *success, NSError **error) {
-				if (boolean == nil) return nil;
-
-				if (![boolean isKindOfClass:NSNumber.class]) {
-					if (error != NULL) {
-						NSDictionary *userInfo = @{
-							NSLocalizedDescriptionKey: NSLocalizedString(@"Could not convert number to boolean-backed number or vice-versa", @""),
-							NSLocalizedFailureReasonErrorKey: [NSString stringWithFormat:NSLocalizedString(@"Expected an NSNumber, got: %@.", @""), boolean],
-							MTLTransformerErrorHandlingInputValueErrorKey : boolean
-						};
-
-						*error = [NSError errorWithDomain:MTLTransformerErrorHandlingErrorDomain code:MTLTransformerErrorHandlingErrorInvalidInput userInfo:userInfo];
-					}
-					*success = NO;
-					return nil;
-				}
-				return (NSNumber *)(boolean.boolValue ? kCFBooleanTrue : kCFBooleanFalse);
-			}];
+		MTLValueTransformer *booleanValueTransformer = [MTLValueTransformer transformerUsingForwardBlock:validateBoolean reverseBlock:validateBoolean];
 
```

Comment:
Strictly speaking, this should be reverted. I hate the "reverse" transformers, but they're still available, so